src/plugin.js

```diff
@@ -12,8 +12,7 @@ var zoomNS = Chart.Zoom = Chart.Zoom || {};
 var zoomFunctions = zoomNS.zoomFunctions = zoomNS.zoomFunctions || {};
 var panFunctions = zoomNS.panFunctions = zoomNS.panFunctions || {};
 
-// Default options if none are provided
-var defaultOptions = zoomNS.defaults = {
+Chart.defaults.global.plugins.zoom = {
```

Comment:
I think users used to configure defaults via `Chart.Zoom.defaults`, so I think you need to keep it working until v1.

```javascript
Chart.Zoom.defaults = Chart.defaults.global.plugins.zoom = {
  // ...
}
```

Reply:
done